docs/reference/transform/overview.asciidoc

```diff
@@ -72,4 +72,10 @@ If your {transform} must process a lot of historic data, it has high resource
 usage initially--particularly during the first checkpoint.
 
 For better performance, make sure that your search aggregations and queries are 
-optimized and that your {transform} is processing only necessary data.
+optimized and that your {transform} is processing only necessary data. Consider 
+whether you can apply a source query to the {transform} to reduce the scope of 
+data it processes. Also consider whether the cluster has sufficient resources in 
+place to support both the composite aggregation search and the indexing of its
+results. You can also throttle the rate at which {transforms} perform search and 
+index requests by setting `docs_per_second` when you <<put-transform,create>> or
```

Comment:
> and index requests

Is this correct that indexing is also affected by this setting?

Reply:
yes, by slowing down the searches, you slow down indexing, too. 

Maybe add a sentence that the transform will take longer if throttling is used, although this seems obvious?

and maybe something like:

To know the current `docs_per_second` transform is operating you can take a look into stats (-> link to stats) and calculate the rate with:

```
docs_per_second = documents_processed / search_time_in_ms * 1000
```

(^ I plan to add this to the stats endpoint, however not for 7.8)

Reply:
Thanks, I've added that information in a final paragraph.